### ISSUE TITLE
AS-11 X9: Confirm what indexing and partitioning are required

### ISSUE THREAD

**Issue by thomasheritage** [AMWA-TV/AS-11_Overview#97]
AS-11 X9 currently provides two indexing and partitioning options:

* one option suitable for CBE (Constant Bytes per Element) Essence -- this involves putting all the Essence in one partition and preceding it with a comprehensive Index Table
* another option designed for VBE (Variable Bytes per Element) Essence (but can also be used with CBE Essence) -- this involves partitioning the Essence every 1-10 minutes and putting an Index Table Segment after each segment of Essence.

AS-11 X9 Essence will probably ordinarily be VBE.

@ClydeSm -- please advise whether AS-11 X9 needs to have the indexing and partitioning details changed.

**Comment by andyquested** [comment]
What makes this work with a generic X9 and a PBS constrained version?

**Comment by thomasheritage** [comment]
It depends on whether anyone has any specific business requirements regarding indexing and partitioning and exactly what the details of those requirements are.

The "MXF Indexing and Partitioning" Block currently in AS-11 X9 is also in X1, X2, X3 and X4. It is also in AS-11 X7 without the VBE Essence part.

**Comment by matthewmarks-bbc** [comment]
Agreed by email: the only rule enforceable is a maximum partition length of 10 minutes, to cope with splicing together sources without having to re-partition. Choosing and sticking to a particular partition size is to be encouraged.
